### PR TITLE
fix(formatting): issue with owner and classification markings navigat…

### PR DESCRIPTION
**Pull request by ola300**
Hey @skhtet / @mleeBoeing / @flesher1,
Please check out my latest changes and thumbs up or down if you have recommendations.  Thanks!

Ola

### CODE REVIEW

**Comment by mleeBoeing** [comment]
@ola300 - You can simply add `ozp-developers` on the right hand side under 'Reviewers' and it'll add the team for review. It's probably easier and faster to do it this way.

**Comment by ola300** [comment]
Thanks! @mleeBoeing  Noted and will do on the next ticket commit/merge.

File: src/styles/main.scss [hunk 1]
```diff
@@ -237,9 +237,10 @@ h5 {
 
 @media (max-width: 1775px) {
     .listingOverview_sidebar {
-        position: inherit !important;
-        display: flex;
-        width: 100% !important;
+        position: absolute;
+        display: none;
```
Comment (mleeBoeing) [inline src/styles/main.scss]: Why is display set to none?

**Comment by flesher1** [comment]
Good start, but a few issues:
the navigation pane and the security marking/owners pane should disappear at the same time. I shouldnt see one without the other.
when going from small window to large window, the sm/owners pane comes back but can cover the navigation pane if the user scrolled down, then back to the top of the page before making the window large again.
Finally, the code you removed wasn't necessary anymore, however you added unnecessary code. You should re-evaluate the code as it is.
For now, :-1:

File: src/styles/main.scss (hunk 1 above)
Comment (ola300) [reply src/styles/main.scss]: @mleeBoeing this was a temporary fix that worked.  So I used this change as a baseline.  I'm gonna show you the latest change using the visibility: hidden feature today.  Hope this'll solve the disappearing of the second nav bar below the Security Markings.

**Comment by ola300** [comment]
@aml-development/ozp-developers ready for review.  Made the necessary changes after reviewing comments from @flesher1 the following fixes are in place after the last pull request.  

**See below:**
**-Modified Media Query settings for sidebar in main.scss (Security Markings | Owner)
-Changed position for listingOverview_sidebar to "fixed" / display: none remains (working) 
-Adjusted navigation bar media query settings**

**Comment by flesher1** [comment]
Several things:

- `display: none;` is not wrong, but you need to understand what it is doing. If you are setting something not to display, consider what else may or may not need to be included into styling.
- you changed it so that the navigation disappears at the same time as Overview sidebar, but it may be disappearing too early, i would do the opposite and make it so that Overview sidebar disappears at the same time as navigation.
- when going from small window to large window, the sm/owners pane comes back but can cover the navigation pane if the user scrolled down, then back to the top of the page before making the window large again.

**Comment by ola300** [comment]
Hey @flesher1 what do you mean by "it may be disappearing too early"?  Also you mention that you see the navigation disappears at the same time.  Then, after you made recommendation to make it so that Overview sidebar disappears at the same time as navigation.  

I don't understand.  Can you be more specific as to the target issue after minimizing... (do the Security Marking / Overview elements remain/disappear separately as before in IE, Firefox, Chrome when minimized or do they disappear together as desired)

**Comment by ola300** [comment]
Also for **display: none** - this code implementation was used to hide both elements (sidebar/navbar) at a specific resolution (i.e example 1775px).  Instead of using visibility: hidden, I kept styling to display: none so the page would fit as desired without compromising the rest of the layout.  This solution was implemented to achieve the current layout without affecting the layout of the similar listings on the right of the page.  Thus, I went ahead and adjusted the media screen min-max pixels so it can fit on multiple browsers (if not mobile devices).

**Comment by ola300** [comment]
@aml-development/ozp-developers fixed padding on "Security Marking | Owner" sidebar.  
This fix resolves overlapping after the user scrolls down pages rendered after selecting listings from the featured listings.

**Comment by flesher1** [comment]
Closing for now.

**Pull request closed without merging**